app/src/main/java/org/fossasia/phimpme/gallery/activities/SecurityActivity.java

```diff
@@ -270,9 +270,9 @@ private void setPasswordDialog() {
             }
 
             @Override public void afterTextChanged(Editable editable) {
-                if(editable.length() == 11) {
-                    editTextConfirmPassword.setText(editable.toString().substring(0, 10));
-                    editTextConfirmPassword.setSelection(10);
+                if(editable.length() == 128) {
```

Comment:
This checks if the total characters entered are exactly 128, it then shows a pop-up displaying maximum password length. But what if someone pastes a password which is greater than 128 characters, then this statement would never be called, and the toast would never be shown.

Reply:
@frextrite If someone pastes a password which is more than 128 characters long, the result would be the same. A toast is displayed and only the first 128 characters of the pasted text are shown.

Reply:
That's great then!